SPECS/xs-openssl.spec

```diff
@@ -532,7 +574,7 @@ export LD_LIBRARY_PATH
 - *** Upstream changelog ***
 - * Wed Mar 23 2022 Clemens Lang <cllang@redhat.com> - 1:1.1.1k-6
 - - Fixes CVE-2022-0778 openssl: Infinite loop in BN_mod_sqrt() reachable when parsing certificates
-- - Resolves: rhbz#2067144
+- - Resolves: rhbz#2067146
```

Comment:
Any idea why this changed? Was the previous reference wrong? Same question for the other similar change.

Reply:
Not really, they do change them in a lot of update. I would assume `rhbz` stands for redhat bugzilla (which as far as I can tell we do not have access to), and they have an internal process where they clone tickets for further version, and somehow update the references to newer tickets for newer versions. I was wondering if we wanted to integrate these changes or not. So I actually integrated them so we can decide this together.

Reply:
I forgot to add that I personally would not integrate them, less changes in the diff, clearer patch, but maybe more diff when updating further.

Reply:
Let's keep as you did, synced with the upstream specfile. I prefer a larger diff now over a larger diff later.